packages/renderer/src/lib/feedback/SendFeedback.svelte

```diff
@@ -71,25 +71,33 @@ async function sendFeedback(): Promise<void> {
           <button aria-label="very-sad-smiley" on:click="{() => selectSmiley(1)}">
             <Fa
               size="1.5x"
-              class="cursor-pointer {smileyRating === 1 ? 'text-violet-400' : 'text-gray-900'}"
+              class="cursor-pointer {smileyRating === 1
+                ? 'text-[var(--pd-invert-content-button-active)]'
```

Comment:
I think using some of the --pd-button colors would be better here, but since this is better and wasn't the focus I'm ok with it.

Reply:
@deboer-tim I updated the colors to use `--pd-button-primary-bg` and `--pd-button-disabled-text`